app/ui/src/main/res/values-zh-rTW/strings.xml

```diff
@@ -148,7 +148,7 @@
   <string name="special_mailbox_name_archive_fmt"><xliff:g id="folder">%s</xliff:g>（歸檔）</string>
   <string name="special_mailbox_name_spam_fmt"><xliff:g id="folder">%s</xliff:g>（垃圾郵件）</string>
   <string name="send_failure_subject">部分郵件沒有成功寄送</string>
-  <string name="debug_version_fmt">版本：<xliff:g id="version">%s</xliff:g></string>
+  <string name="version">版本：<xliff:g id="version">%s</xliff:g></string>
```

Comment:
Interpolation symbols are not removed here as they are in english version resulting in "Version: %s" (where "Version" is localized and `%s` is printed verbatim).

I'm not entirely sure how should these strings be handled w.r.t. Transifex.

Reply:
Thanks for spotting, fixed in b9724b4
I've never used Transifex before, though [the docs](https://docs.transifex.com/formats/android-xml/) suggest the files can be re-imported